tensorflow/compiler/xla/service/gpu/gemm_thunk.cc

```diff
@@ -288,6 +293,12 @@ Status RunGemm(const GpuGemmConfig &gemm_config,
             batch_size, lhs_matrix, rhs_matrix, output_matrix, alpha.real(),
             beta, stream, best_algorithm,
             /*output_profile_result=*/profile_result);
+      case CUS:
+        CHECK_EQ(alpha.imag(), 0);
+        return DoGemmWithAlgorithm<cus, double>(
```

Comment:
why is it cus and double? What are these types used for?

Reply:
Resolved in the chat